tests/unit/test_decorator.py

```diff
@@ -33,6 +33,10 @@
 
 
 class MyProcessor(Processor):
+    @classmethod
+    def get_default_error_builder(cls) -> "ErrorBuilderInterface":
```

Comment:
is it possible to avoid typing defined as string?

Reply:
We have an cyclic import:

```
tests/ext/unit/test_bottle.py:5: in <module>
    import hapic
hapic/__init__.py:2: in <module>
    from hapic.hapic import Hapic
hapic/hapic.py:9: in <module>
    from hapic.context import ContextInterface
hapic/context.py:6: in <module>
    from hapic.error.main import ErrorBuilderInterface
hapic/error/main.py:6: in <module>
    from hapic.processor.main import ProcessValidationError
hapic/processor/main.py:11: in <module>
    from hapic.error.main import ErrorBuilderInterface
E   ImportError: cannot import name 'ErrorBuilderInterface' from 'hapic.error.main' 
```

Reply:
I was wrong, cyclic import is here `hapic.processor.main.Processor#get_default_error_builder` not where you ask. A changed it.